backend/Application/Aggregates/Contract/Jobs/ContractDatabaseImportJob.cs

```diff
@@ -74,6 +85,27 @@ public string GetUniqueIdentifier()
         return JobName;
     }
 
+    private async Task UpdateReadHeightMetric(CancellationToken token)
+    {
+        try
+        {
+            while (!token.IsCancellationRequested)
+            {
+                var repository = await _repositoryFactory.CreateAsync();
+                var latest = await repository.GetReadOnlyLatestContractReadHeight();
+                if (latest != null)
+                {
+                    ContractMetrics.SetReadHeight(latest.BlockHeight, ImportSource.DatabaseImport);
+                }
+
+                await Task.Delay(_contractAggregateOptions.MetricDelay, token);
+            }
+        }
+        catch (TaskCanceledException)
+        {
+        }
```

Comment:
We should at the very least have a code comment explaining why it's empty, despite it seeming obvious from the context.